src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/System/Windows/Automation/AutomationHeadingLevel.cs

```diff
@@ -18,7 +18,7 @@ public enum AutomationHeadingLevel
         /// <summary>
         /// The element does not have a heading level
         /// </summary>
-        None = 80050,
+        None = 1,
```

Comment:
```suggestion
        None = 0,
```

So that HeadingLevel1 = 1, etc.  I think this means markup can say `<TextBlock AutomationProperties.HeadingLevel="1">` and the right thing will happen.

Reply:
`<TextBlock AutomationProperties.HeadingLevel="1">`
When I tried this markup here in my local test, it raises an error saying that numbers are not valid as enumeration values: 

```
(Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0431	Invalid value for property 'AutomationProperties.HeadingLevel': '1'	test	C:\ ....	
```
It forces us to complete with the enum value ("None", "Level1", etc).

Reply:
OK.  But I still like None=0 (and Level1=1), if only because it looks better.  (It also agrees with what I wrote in the spec.)